resolver/ng/map.jinja

```diff
@@ -0,0 +1,33 @@
+{% import_yaml "resolver/ng/defaults.yaml" as cfg %}
+
+# Main defaults.
+{% set defaults = cfg.Defaults %}
+
+# os_family grain vars.
+{% set os_family_vars = salt['grains.filter_by'](
+    cfg,
```

Comment:
Would prefer to see OS explicitly listed here rather than in defaults.yaml and pillar.example.

Reply:
I believe that defaults should go to `defaults.yaml `, and the mapping of the data and dynamic values done inside `map.jinja`.

I really find it not good at all to deal with static data inside `map.jinja`.
I believe it could be moved to (osmap.yaml](https://github.com/search?q=org%3Asaltstack-formulas+osmap.yaml&type=Code&utf8=%E2%9C%93) (or so) but not to `map.jinja`.

Reply:
Thank you!  On recent PR, I assumed template-formula was a consensus but a maintainer argued against over complicating ..  https://github.com/saltstack-formulas/samba-formula/pull/28#discussion_r146082431 and I now share that opinion (not strongly) that map.jinja is best - this is implementation detail however.